lib/inputs/dovetail-impressions.js

```diff
@@ -38,46 +36,27 @@ module.exports = class DovetailImpressions {
   }
 
   format(record) {
-    let lookups = [lookip.look(record.remoteIp), lookagent.look(record.remoteAgent)];
-    return Promise.all(lookups).then(([geo, agent]) => {
-      return {
-        table: this.table(record),
-        record: {
-          // TODO: this ASSUMES any ad is only found once per arrangement!!!
-          insertId: this.md5(record),
-          json: {
-            digest:         record.digest,
-            program:        record.program,
-            path:           record.path,
-            feeder_podcast: record.feederPodcast,
-            feeder_episode: record.feederEpisode,
-            remote_agent:   record.remoteAgent,
-            remote_ip:      record.remoteIp,
-            timestamp:      timestamp.toEpochSeconds(record.timestamp || 0),
-            request_uuid:   record.requestUuid,
-            ad_id:          record.adId,
-            campaign_id:    record.campaignId,
-            creative_id:    record.creativeId,
-            flight_id:      record.flightId,
-            is_duplicate:   record.isDuplicate,
-            cause:          record.cause,
-            city_id:        geo.city,
-            country_id:     geo.country,
-            agent_name_id:  agent.name,
-            agent_type_id:  agent.type,
-            agent_os_id:    agent.os
-          }
+    return Promise.resolve({
+      table: 'dt_impressions',
+      record: {
+        // TODO: this ASSUMES any ad is only found once per arrangement!!!
+        insertId: this.md5(record),
+        json: {
+          timestamp:      timestamp.toEpochSeconds(record.timestamp || 0),
+          request_uuid:   record.requestUuid,
```

Comment:
are you able to leave out `agent_` and `remote_` attributes b/c they are in downloads?

Reply:
Yep - can always `JOIN dt_downloads d ON (i.request_uuid = d.request_uuid)`.  A tiny bit slower, but I think it's fine since we rarely need to know those properties of an impression.